src/flexible_type/flexible_type.hpp

```diff
@@ -281,7 +282,14 @@ class flexible_type {
    */
   flexible_type(flexible_type& other) noexcept;
 
+  template <size_t N>
+  flexible_type(const char v[N]):flexible_type() {
```

Comment:
might be nice to add the `void` return type to these functions.

Reply:
This is a constructor.

Reply:
make sense